### PR TITLE
Feature: "Creature Harvesting"

### PR DESCRIPTION
**Pull request by cbortz**
## tl;dr

- Allows users to view creature resource yields from a higher level (_see: `creatureList.py`_)
- Refactors `getCreatureList.py` to query against multiple columns
- _Fix_: Python 2.7 string formatting syntax error

---

## Reasoning
As a user looking for information on creature resources, I am able to see the creatures that drop a given resource (if the information is available):

previous

Sometimes, however, I don't care about a specific quality or granularity. So instead of finding which creature yields the most _Tatooinian Leathery Hide_, I just want to know which creature yields the most of any _hide_ (or meat, or bone).

**Since the information was already in the database, I built it.** Here are some screen shots:

harvesting

bone

sidebar

---

_**Note:** I've not written Python in quite some time, but I tried to follow the project's structure as close as possible._

### CODE REVIEW

**Comment by cbortz** [comment]
Hey @pwillworth I'm not attached to the name of the feature at all. This is just something that I'd love to see implemented. Cheers!
